server/studyspaces.py

```diff
@@ -59,32 +38,34 @@ def parse_times(building):
         start = request.args.get('start')
         end = request.args.get('end')
 
-    try:
-        rooms = studyspaces.get_rooms(building, start, end)
-
-        # legacy support for old scraping method
-        rooms["location_id"] = rooms["id"]
-        rooms["rooms"] = []
-        for room_list in rooms["categories"]:
-            for room in room_list["rooms"]:
-                room["thumbnail"] = room["image"]
-                del room["image"]
-                room["room_id"] = room["id"]
-                del room["id"]
-                room["gid"] = room_list["cid"]
-                room["lid"] = building
-                room["times"] = room["availability"]
-                del room["availability"]
-                for time in room["times"]:
-                    time["available"] = True
-                    time["start"] = time["from"]
-                    time["end"] = time["to"]
-                    del time["from"]
-                    del time["to"]
-                rooms["rooms"].append(room)
-    except APIError as e:
-        return jsonify({"error": str(e)})
-
+    if building == 1:
+        sessionid = get_sessionid();
+        print(sessionid)
```

Comment:
Remove semicolon and print statements before merging. You can use `flake8` to check for style issues before pushing.